mcf-browser/src/back-end/server-api.js

```diff
@@ -54,11 +54,15 @@ async function readFileList(fileList) {
             const mcfParser = new ParseMcf(tmcf.name + '&' + file.name);
             return mcfParser.parseMcfStr(mcf);
           });
-        finalReturn['errMsgs'] =
+
+        if(tmcfOut['errMsgs'] && tmcfOut['errMsgs'].length !== 0) {
```

Comment:
Just wondering why just having the second condition may not work?  Given https://github.com/datacommonsorg/tools/blob/4bd98c15a4fd309ece98301da4946234da24b61a/mcf-browser/src/back-end/parse-mcf.js#L69